client/landing/gutenboarding/components/domain-categories/style.scss

```
@@ -1,5 +1,47 @@
 @import '../../mixins';
 
+.domain-categories {
+
+	@media ( max-width: $break-medium ) {
+		margin-bottom: 20px;
+
+		.domain-categories__dropdown-button.components-button {
+			display: block;
+		}
+
+		.domain-categories__item-group {
+			display: none;
+		}
+
+		&.is-open {
+			.domain-categories__item-group {
+				display: block;
+			}
+		}
+	}
+}
+
+.domain-categories__dropdown-button.components-button {
+	width: 100%;
+	text-align: center;
+	margin-bottom: 8px;
+	height: 40px;
+	border: 1px solid var( --studio-gray-5 );
+	display: none;
+
+	svg {
+		margin-left: 5px;
+	}
+}
```

I think it would look better if we remove the bottom margin. And also the bottom border when the dropdown is open.

```suggestion
.domain-categories__dropdown-button.components-button {
	width: 100%;
	text-align: center;
	height: 40px;
	border: 1px solid var( --studio-gray-5 );
	display: none;

	svg {
		margin-left: 5px;
	}
}

// hide bottom border when the dropdown is open
.domain-categories.is-open .domain-categories__dropdown-button.components-button {
	border-bottom: none;
}
```